harness/determined/common/experimental/determined.py

```diff
@@ -255,3 +255,33 @@ def get_model_labels(self) -> List[str]:
         Get a list of labels used on any models, sorted from most-popular to least-popular.
         """
         return list(bindings.get_GetModelLabels(self._session).labels)
+    
+    def list_clients(self) -> Sequence[oauth2_scim_client.Oauth2ScimCient]:
```

Comment:
i think we should namespace this somewhere other than the det object? cc @rb-determined-ai

Reply:
oh yes that's true. I can call them list_oauth2_scim_clients(), remove_oauth2_scim_client(), add_oauth2_scim_client()?

Reply:
okay leaving this in determined because there are were plans to namespace things outside determined.